__tests__/app.test.js

```diff
@@ -96,3 +98,48 @@ describe("get /api/topics", () => {
       });
   });
 });
+
+describe("test4.get /api/articles/:article_id", () => {
+  test("200: responds with an object of article id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        // console.log(response);
+
+        //   expect(message).toBeInstanceOf(Object);
+        // expect(testStack.hasOwnProperty("quantity")).toBe(true);
+
+        expect(typeof response).toBe("object");
+        expect.objectContaining({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+        });
+      });
+  });
+});
+
+describe("test4.get /api/articles/:article_id", () => {
+  test("should respond with an correct article id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        // console.log(body);
+        expect(body).toEqual({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 100,
+        });
+      });
+  });
+});

```

Comment:
Have you though about error handling here? What happens if the id does not exist yet or if it is the wrong format like api/articles/a?